release/ray_release/wheels.py

```diff
@@ -150,6 +150,32 @@ def find_and_wait_for_ray_wheels_url(
     return wait_for_url(ray_wheels_url, timeout=timeout)
 
 
+def get_buildkite_repo_branch() -> Tuple[str, str]:
+    if "BUILDKITE_BRANCH" not in os.environ:
+        return DEFAULT_REPO, DEFAULT_BRANCH
+
+    branch_str = os.environ["BUILDKITE_BRANCH"]
```

Comment:
should there be a DEFAULT_BRANCH given you had it in current implementation ?

Reply:
With line 154 we return the DEFAULT_REPO + BRANCH if it is not set, line 157 is only reached when the env variable is set